Sources/protoc-gen-swift/FileGenerator.swift

```diff
@@ -259,10 +267,19 @@ class FileGenerator {
             p.print("\n")
         }
 
-        p.print(
-            "import Foundation\n",
-            "import SwiftProtobuf\n",
-            "\n")
+        // The well known types don't need to import SwiftProtobuf, so catch
+        // them via their package (just make sure it isn't descriptor since that
+        // isn't needed/shipped, it would be outside the library.
+        if isWellKnownType {
+          p.print(
+              "import Foundation\n",
```

Comment:
Maybe move the `import Foundation` line out above this conditional, so that if we ever decided to add more imports (I'm not sure what they would be though), we avoid duplication?